packages/components/droppable-panel/src/droppable-panel.styles.js

```diff
@@ -0,0 +1,13 @@
+import styled from '@emotion/styled';
+import { customProperties as vars } from '@commercetools-uikit/design-system';
+
+const TagContainerEditable = styled.div`
+  background-color: ${vars.backgroundColorForInput};
+  border: ${vars.borderRadius1} solid #999;
```

Comment:
We should use the colors from our design tokens. There should be an available colour that corresponds to this one, since I don't think it's justifiable to add a new one.

Reply:
I'll check the one that is closest

Reply:
`colorNeutral60` same color in hsl format